Directory.Build.props

```diff
@@ -1,34 +1,26 @@
-<?xml version="1.0" encoding="utf-8"?>
 <Project>
-    <Import Project="Sdk.props" Sdk="Microsoft.DotNet.Arcade.Sdk" />
-    <Import Project="$(RepositoryEngineeringDir)Analyzers.props" />
 
-    <PropertyGroup>
-        <LangVersion>preview</LangVersion>
-        <NETCoreTargetFramework>net8.0</NETCoreTargetFramework>
-        <NETStandardTargetFramework>netstandard2.0</NETStandardTargetFramework>
-        <NETFullTargetFramework>net472</NETFullTargetFramework>
```

Comment:
imo removing those properties is not for good. 
They name intention correctly and I prefer to keep them, also for .NET Standard.

They may be changed as
```
        <NETTargetFramework>$(NetCurrent)</NETTargetFramework>
        <NETStandardTargetFramework>netstandard2.0</NETStandardTargetFramework>
        <NETFullTargetFramework>net472</NETFullTargetFramework>
```
to remove `Core` and make use of automatic .NET (core) updates.